### PR TITLE
feat: Add firmware version to diagnostics

### PR DESCRIPTION
**Pull request by YannickdeHoop**

### CODE REVIEW

File: include/sick_safetyscanners/SickSafetyscanners.h [hunk 1]
```diff
@@ -132,12 +132,18 @@ class SickSafetyscanners
   void requestConfigMetadata(const datastructure::CommSettings& settings,
                              datastructure::ConfigMetadata& config_metadata);
 
+  /*!
```
Comment (reinzor) [inline include/sick_safetyscanners/SickSafetyscanners.h:135]: why is this moved?

Comment (YannickdeHoop) [reply include/sick_safetyscanners/SickSafetyscanners.h:135]: all implemented functions are moved upwards with docstrings ...

**Review by reinzor — APPROVED** [review]
